quantum/process_keycode/process_tap_dance.h

```diff
@@ -34,7 +34,7 @@ typedef struct
   bool finished;
 } qk_tap_dance_state_t;
 
-#define TD(n) (QK_TAP_DANCE + n)
+#define TD(n) (QK_TAP_DANCE | ((n) & 0xFF))
```

Comment:
yeah probably should just define #QK_MAX_TAP_DANCE 0xFF so people understand what it is.

Reply:
God naming is hard.

Reply:
What about things like `LT()`? `QK_LAYER_TAP_MAX` is just the max value of the Quantum keycode, but it takes two arguments - the layer and the tap keycode.

Reply:
I've just seen the other instances of `0xff` within `quantum_keycodes.h`, maybe it could be cleaner to make that request in another PR and limit the scope of this one.

Reply:
Yep, ok with that. Just don't like magic numbers either.